To: vim_dev@googlegroups.com
Subject: Patch 7.3.182
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.182 (after 7.3.180)
Problem:    Compiler warning for uninitialized variable.
Solution:   Add dummy initializer.
Files:	    src/misc1.c


*** ../vim-7.3.181/src/misc1.c	2011-05-10 13:38:23.000000000 +0200
--- src/misc1.c	2011-05-10 14:37:39.000000000 +0200
***************
*** 1563,1569 ****
      char_u	*list;
      int		middle_match_len = 0;
      char_u	*prev_list;
!     char_u	*saved_flags;
  
      i = 0;
      while (vim_iswhite(line[i]))    /* leading white space is ignored */
--- 1563,1569 ----
      char_u	*list;
      int		middle_match_len = 0;
      char_u	*prev_list;
!     char_u	*saved_flags = NULL;
  
      i = 0;
      while (vim_iswhite(line[i]))    /* leading white space is ignored */
*** ../vim-7.3.181/src/version.c	2011-05-10 14:22:10.000000000 +0200
--- src/version.c	2011-05-10 14:38:39.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     182,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
75. You start wondering whether you could actually upgrade your brain
    with a Pentium Pro microprocessor 80.  The upgrade works just fine.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
